The following code will throw an error, because the function 'foo' includes two parameters, but is only invoked with one argument.